schema/home/index.js

```diff
@@ -8,8 +10,10 @@ import {
 const HomePageType = new GraphQLObjectType({
   name: 'HomePage',
   fields: {
-    modules: HomePageModules,
```

Comment:
I don't think this is a breaking change, since Force directly requests `home_page_modules` and `home_page_module`, and doesn't ask it via `home { ...`